app/src/lang/translations/en-US.yaml

```
@@ -1364,6 +1364,16 @@ fields:
     custom_aspect_ratios: Custom Aspect Ratios
     theme_light_overrides: Light Theme Customization
     theme_dark_overrides: Dark Theme Customization
+    public_registration: User Registration
+    public_registration_note: Allows users to register via the [registration route](/admin/register).
```

Maybe we can use technical wording? 

```suggestion
    public_registration_note: Allows users to register via a [registration page](/admin/register).
```

---

Additionally, I'd probably leave out the link. Maybe something like this?
```suggestion
    public_registration_note: Allows users to self-register to the project.
```

---

Linking is useful because not everyone will know where the registration will appear or how. Why would less information here be better than linking to the page?

---

The feature won't be limited to the register page of the Data Studio, but also for registering via API, so a more general note seems appropriate to me 👍

---

> The feature won't be limited to the register page

Ahh true, good point! But I still like linking to it, I regard that as useful UX for admins. I used it quite often. mhh

---

Maybe a combination of both then?
```suggestion
    public_registration_note: Allows users to register via the [registration page](/admin/register) and API.
```

```suggestion
    public_registration_note: Allows users to self-register via the [registration page](/admin/register) and API.
```